packages/desktop-client/src/components/reports/ReportOptions.ts

```diff
@@ -294,8 +294,9 @@ export const categoryLists = (categories: {
   list: CategoryEntity[];
   grouped: CategoryGroupEntity[];
 }) => {
+  const categoriesToSort = [...categories.list];
```

Comment:
Make a copy before sorting so that `.sort` doesn't do an in-place sort directly on the `categories.list` state